### PR TITLE
LPD-40180 Use ExternalReferenceCodes instead of id on Organization Management widget configuration

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-40180

### CODE REVIEW

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:upgrade

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:upgrade

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

File: modules/dxp/apps/commerce/commerce-organization-web/src/main/java/com/liferay/commerce/organization/web/internal/display/context/CommerceOrganizationDisplayContext.java [hunk 1]
```diff
@@ -41,10 +42,12 @@ public class CommerceOrganizationDisplayContext {
 
 	public CommerceOrganizationDisplayContext(
 			HttpServletRequest httpServletRequest,
+			OrganizationLocalService organizationLocalService,
 			OrganizationService organizationService,
 			UserLocalService userLocalService)
 		throws PortalException {
 
+		_organizationLocalService = organizationLocalService;
```
Comment (alessiorendina) [inline modules/dxp/apps/commerce/commerce-organization-web/src/main/java/com/liferay/commerce/organization/web/internal/display/context/CommerceOrganizationDisplayContext.java:50]: Remove `_themeDisplay` variable and use `_commerceOrganizationRequestHelper` instead.

Also, pass ConfigurationProvider to constructor instead.

Comment (alessiorendina) [reply modules/dxp/apps/commerce/commerce-organization-web/src/main/java/com/liferay/commerce/organization/web/internal/display/context/CommerceOrganizationDisplayContext.java:50]: Same for PortalUtil, pass Portal to constructor instead.

File: modules/dxp/apps/commerce/commerce-organization-test/build.gradle [hunk 2]
```diff
@@ -0,0 +1,20 @@
+dependencies {
+	testIntegrationImplementation group: "com.liferay.portal", name: "com.liferay.portal.impl", version: "default"
+	testIntegrationImplementation group: "javax.portlet", name: "portlet-api", version: "3.0.1"
+	testIntegrationImplementation group: "org.apache.felix", name: "org.apache.felix.http.servlet-api", version: "1.1.2"
+	testIntegrationImplementation group: "org.osgi", name: "org.osgi.service.cm", version: "1.6.0"
+	testIntegrationImplementation project(":apps:asset:asset-link-api")
+	testIntegrationImplementation project(":apps:export-import:export-import-api")
+	testIntegrationImplementation project(":apps:export-import:export-import-test-util")
+	testIntegrationImplementation project(":apps:frontend-taglib:frontend-taglib-form-navigator")
```
Comment (alessiorendina) [inline modules/dxp/apps/commerce/commerce-organization-test/build.gradle:9]: Check if this one is necessary

File: modules/dxp/apps/commerce/commerce-organization-test/src/testIntegration/java/com/liferay/commerce/organization/web/internal/exportImport/test/CommerceOrganizationExportImportTest.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+		Organization organization = OrganizationTestUtil.addOrganization();
+
+		PortletPreferences portletPreferences = getImportedPortletPreferences(
+			HashMapBuilder.put(
+				"rootOrganizationExternalReferenceCode",
+				() -> new String[] {organization.getExternalReferenceCode()}
+			).build());
+
+		Organization importedOrganization =
+			OrganizationLocalServiceUtil.
```
Comment (alessiorendina) [inline modules/dxp/apps/commerce/commerce-organization-test/src/testIntegration/java/com/liferay/commerce/organization/web/internal/exportImport/test/CommerceOrganizationExportImportTest.java:81]: Use `_organizationLocalService` instead

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@monaco-fabio LGTM overall, just few small changes. Thank you!

**Comment by monaco-fabio** [comment]
ci:report:85173698

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:upgrade

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
@monaco-fabio confirmed that failures are not related. I'll forward it, thanks!

**Comment by alessiorendina** [comment]
ci:forward:force

**Pull request closed without merging**